### PR TITLE
Brian's pull request

### PR DESCRIPTION
**Pull request by brian-lo0-127001**
This pull request is created to address the code review standards.

### CODE REVIEW

File: cst438-week2/.gitignore [hunk 1]
```diff
@@ -6,9 +6,7 @@ target/
 
 ### STS ###
 .apt_generated
-.classpath
```
Comment (brian-lo0-127001) [inline cst438-week2/.gitignore:9]: This change was needed for Github interoperability with Eclipse.

File: cst438-week2/.gitignore [hunk 2]
```diff
@@ -6,9 +6,7 @@ target/
...
 .factorypath
-.project
```
Comment (brian-lo0-127001) [inline cst438-week2/.gitignore:11]: This was necessary for Eclipse.

**Comment by Scorp197** [comment]
Your pull requests worked.

**Review by brian-lo0-127001 — COMMENTED** [review]
This review is done.

File: cst438-week2/.gitignore (hunk 2 above)
Comment (Scorp197) [reply cst438-week2/.gitignore:11]: Good. Now I can safely import your repo to Eclipse

File: cst438-week2/.gitignore (hunk 1 above)
Comment (Scorp197) [reply cst438-week2/.gitignore:9]: Good change.

**Pull request merged**